required/latex-lab/documentmetadata-support.dtx

```diff
@@ -137,37 +137,39 @@
 %      The |u| variants do not enforce unicode,
 %      but they will pass the information to hyperref. The |a| variants
 %      do \emph{not} enforce (or even test) a tagged pdf yet.
-%      
-%      Starting with version 0.95s of \pkg{pdfmanagement-testphase}
-%      it is also possible to use the values
-%      |X-4|, |X-4p|, |X-5g|, |X-5n|, |X-5pg|, |X-6|, |X-6n|, |X-6p|, |UA-1| for
-%      a PDF/X and PDF/UA standard. These keys currently set \emph{only} the relevant
-%      XMP-metadata. In version 0.95z support for |UA-2| has been added but
-%      note that |UA-2| hasn't been released yet. It should be used only together
-%      with pdf version 2.0.
+%
+%      Beside the A-standards it is also possible to use the values
+%      |X-4|, |X-4p|, |X-5g|, |X-5n|, |X-5pg|, |X-6|, |X-6n|, |X-6p| for
+%      a PDF/X and |UA-1|  and |UA-2| for PDF/UA standard.
+%      |UA-2| should only be used together with PDF 2.0.
+%      These keys currently set mostly \emph{only} the relevant
+%      XMP-metadata.
 %
 %      |pdfstandard| can be used more than once to set overlapping standards, e.g:\\
 %      |pdfstandard=A-2b,pdfstandard=X-4,pdfstandard=UA-1|
-%      
+%
 %      If XMP-metadata are added (see the following key \texttt{xmp})
 %      the needed conformance marker for the standards are set.
```

Comment:
needed -> necessary

Reply:
Also: markers . . . are 
Or: marker . . . is